__tests__/app.test.js

```diff
@@ -91,7 +91,30 @@ describe("GET/api/articles", () => {
           });
         });
         expect(articles).toBeSortedBy("created_at", { descending: true });
-        console.log(articles);
+      });
+  });
+});
+
+describe("GET/api/articles/:article_id/comments", () => {
+  it("get all comments for an article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
+          expect(comment.article_id).toBe(1);
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+          });
```

Comment:
You are missing several tests here. In this test you aren't checking that the order returned is correct.

You are also missing:

- an article with no comments, what status code would this be?
- an article that doesn't exist, eg 99
- an article id that is invalid

Reply:
I have added a few more tests based on your suggestion.